README.md

```diff
@@ -212,9 +218,25 @@ const contract = new HRC20('0x...00', ABI, wallet, {
 
 ### Methods
 
-#### balanceOf
+#### totalSupply(): Promise&lt;BN&gt;
+
+<p>Returns the amount of tokens in existence.</p>
+
+```ts
+import { HttpProvider } from '@harmony-js/network'
+import { HRC20 } from 'harmony-marketplace-sdk'
+import * as ABI from './abi.json'
+
+// A contract instance
+const contract = new HRC20('0x...00', ABI, HttpProvider)
```

Comment:
This should be:
`
const wallet = new PrivateKey(
  HarmonyShards.SHARD_0,
  '45e497bd45a9049bcb649016594489ac67b9f052a6cdf5cb74ee2427a60bf25e'
)
// A contract instance
const contract = new HRC20('0x...00', ABI, wallet)
`

Reply:
Fixed in here https://github.com/blockcoders/harmony-marketplace-sdk/pull/54/commits/ee0e67a64f7ef634e193909de7e321080bb84c01